scrape-targets/scrape-targets.json

```diff
@@ -4,8 +4,8 @@
       "job": "nodes"
     },
     "targets": [
-      "localhost:5054",
-      "localhost:5064"
+      "host.docker.internal:5054",
+      "host.docker.internal:5064"
```

Comment:
I can't get this working on Linux. I tried adding `extra_hosts` to the docker-compose.yaml and changing my metrics listen address to `0.0.0.0` on Lighthouse, but it still times out in Prometheus with this error:

> Get "http://host.docker.internal:5054/metrics": context deadline exceeded

I followed the instructions from here: https://stackoverflow.com/questions/48546124/what-is-linux-equivalent-of-host-docker-internal/67158212#67158212

Reply:
I'm using Docker 24.0.6, so I don't think it's a version issue. And I can see `172.17.0.1` in `/etc/hosts` inside the container. It just times out when I make requests to it

Reply:
hmm interesting, I don't have a Linux to test rn but was trying to run this again on my Mac today and reminded me of this branch 😅 

Not sure why but I had to rebuild the images to get Grafana working again for me (the containers run fine but I can't access the UI).. very strange. Although it looks like you're having a different issue. Could it be firewall on your Linux host?

Yeah it's probably best to leave this PR unmerged for now.

Reply:
I don't have a firewall 😳

Reply:
@michaelsproul are you trying to use this on Linux to connect Prometheus from outside of the machine? if then, you dont need to disable network_mode `host`, just update `grafana.ini` and `prometheus/Dockerfile`. thats enough